src/app/utilities/api-clients/datasets.js

```diff
@@ -96,8 +96,10 @@ export default class datasets {
                 return;
             }
 
-            const latestVersionURL = url.resolve(dataset.current.links.latest_version.href);
-            const latestVersion = await http.get(`/dataset${latestVersionURL}`).catch(error => reject(error));
+            const latestVersionID = dataset.current.links.latest_version.id;
```

Comment:
Is it possible that the latest version when calling GET `/dataset/{id}` may return a version for a different edition of the dataset than the one you care about?

Where there are two or more editions to a dataset?

I guess it depends if this method is only used from a dataset page.

Reply:
Should you be calling the edition endpoint? And getting latest version from there instead?

Reply:
A good point - I think it is possible for a version of a different edition to be returned on the dataset endpoint. I guess it's a product question whether we should pull data from the latest version, or the latest version of that edition 🤔

Reply:
I think there could be a possibility to create two distinct methods in code, and both would have use cases on the website, depending on where what functionality you are trying to provide.